fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/FintechConsentUiSmokeE2ETest.java

```diff
@@ -0,0 +1,245 @@
+    public void testEmbeddedMaxMustermanWantsItsAccountsAndTransactionsFromFintech(FirefoxDriver firefoxDriver) {
+        given().enabled_embedded_sandbox_mode(smokeConfig.getAspspProfileServerUri())
+                .fintech_points_to_fintechui_login_page(smokeConfig.getFintechServerUri());
+        when().user_already_login_in_bank_profile(firefoxDriver)
+                .and()
+                .user_max_musterman_provided_to_consent_ui_initial_parameters_to_list_transactions_with_all_accounts_consent(firefoxDriver)
+                .and()
+                .user_max_musterman_in_consent_ui_reviews_transactions_consent_and_accepts(firefoxDriver)
+                .and()
+                .user_max_musterman_in_consent_ui_provides_pin(firefoxDriver)
+                .and()
+                .user_max_musterman_in_consent_ui_sees_sca_select_and_selected_type_email2_to_embedded_authorization(firefoxDriver)
+                .and()
+                .user_max_musterman_in_consent_ui_provides_sca_result_to_embedded_authorization(firefoxDriver)
+                .and()
+                .user_max_musterman_in_consent_ui_sees_thank_you_for_consent_and_clicks_to_tpp(firefoxDriver)
+                .and()
+                .user_navigates_to_page(firefoxDriver)
+                .and()
+                .user_anton_brueckner_in_consent_ui_sees_thank_you_for_consent_and_clicks_to_tpp(firefoxDriver)
+                .and()
+                .user_sees_account_and_list_transactions(firefoxDriver);
+
+        String userId = getInfoId(firefoxDriver);
+        String bankId = getBankInfoId(firefoxDriver);
+
+        then().fintech_can_read_max_musterman_accounts_and_transactions(userId, bankId);
+    }
+
+    @Test
+    void testEmbeddedAntonBruecknerWantsToSeeItsAccountsAndTransanctionsFromFintech(FirefoxDriver firefoxDriver) {
+        given().enabled_embedded_sandbox_mode(smokeConfig.getAspspProfileServerUri())
+                .fintech_points_to_fintechui_login_page(smokeConfig.getFintechServerUri());
+        when().user_already_login_in_bank_profile(firefoxDriver)
+                .and()
+                .user_anton_brueckner_provided_to_consent_ui_initial_parameters_to_list_transactions_with_all_accounts_consent(firefoxDriver)
+                .and()
+                .user_anton_brueckner_in_consent_ui_reviews_transaction_consent_and_accepts(firefoxDriver)
+                .and()
+                .user_anton_brueckner_in_consent_ui_provides_pin(firefoxDriver)
+                .and()
+                .user_anton_brueckner_in_consent_ui_provides_sca_result_to_embedded_authorization(firefoxDriver)
+                .and()
+                .user_anton_brueckner_in_consent_ui_sees_thank_you_for_consent_and_clicks_to_tpp(firefoxDriver)
+                .and()
+                .user_navigates_to_page(firefoxDriver)
+                .and()
+                .user_sees_account_and_list_transactions(firefoxDriver);
+
+        String userId = getInfoId(firefoxDriver);
```

Comment:
You do not need to parse that - you need to click on the user account rectangle using Selenium. Also, this is not userId, but accountId